### DISCUSSION TITLE
Fish Folk & Fish Folks?

### DISCUSSION THREAD

**Discussion by zicklag** [fishfight/punchy#207] [Q&A] [answered]
@erlend-sh I just realized that we put "Fish Folk" in the repo title, but the organization name is `fishfolks`, is that intentional?

It's slightly inconsistent seeming because the org short name is `fishfolks` but the org human title is still just "Fish Folk" without the `s` at the end.

![image](https://user-images.githubusercontent.com/25393315/183263568-4e321c26-0739-4f72-a19a-23ef7e914c18.png)

**Comment by erlend-sh** [comment]
If you go to https://github.com/fishfolk you’ll see that it’s unfortunately being squatted 🤷‍♂️

**Reply by zicklag** [reply]
Oh...bummer. OK, not a big deal, then. :smiley:

**Reply by erlend-sh** [reply]
There’s a process to putting a claim on unused usernames like that, and we’re starting it, but it will take a while.

**Comment by erlend-sh** [comment, marked as answer]
As a result of registering Fish Folk ™️ as a formally recognized trademark, the dormant `fishfolk` name has been granted to us!
